scripts/HealthCheck.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.digital.ho.hocs;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+/**
+ * Class used for container health check by docker compose
+ * Pass address to be health checked as the single command line argument
+ */
+public class HealthCheck {
```

Comment:
case-creator doesn't have any http endpoints. It doesn't have any dependancys so doesn't need the health check.

Reply:
I've removed the file from this repo now